### PR TITLE
Adding UPS 1.0.3 blog

### PR DESCRIPTION
**Pull request by matzew**
Notes on UPS 1.0.3 relase

@sebastienblanc @lfryc can you review and merge (including production branch)?

### CODE REVIEW

**Comment by sebastienblanc** [comment]
:+1:

File: _posts/2015-03-23-aerogear-ups-103.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+The AeroGear team is happy to announce the version 1.0.3 of the UnifiedPush Server! The release focuses on stabilization and adds a few new features
+
+* Migration support for 1.0.x users as explained in our [user guide](https://aerogear.org/docs/unifiedpush/ups_userguide/index/#migration-guide)
+* Keycloak version update to 1.0.5
+* Handing APNs errors with bad iOS device tokens
+* Improvements on internal storage of device metadata
+
+The complete list of included items are avialble on our [JIRA instance](https://issues.jboss.org/browse/AGPUSH/fixforversion/12325082/)
```
Comment (cvasilak) [inline _posts/2015-03-23-aerogear-ups-103.md]: s/avialble/available

**Comment by cvasilak** [comment]
:+1:

**Comment by lfryc** [comment]
Reviewing...

File: _posts/2015-03-23-aerogear-ups-103.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+section: news
+title: AeroGear UnifiedPush Server 1.0.3 released
+author: matzew
+module: push
+releases: 1.0.3
+---
+
+The AeroGear team is happy to announce the version 1.0.3 of the UnifiedPush Server! The release focuses on stabilization and adds a few new features
+
+* Migration support for 1.0.x users as explained in our [user guide](https://aerogear.org/docs/unifiedpush/ups_userguide/index/#migration-guide)
```
Comment (lfryc) [inline _posts/2015-03-23-aerogear-ups-103.md]: Is migration a feature?

If yes, I would start with the features that caused a need for migration and then explained they unfortunately requires database migration.

**Comment by lfryc** [comment]
Except my comment above, it looks great!

File: _posts/2015-03-23-aerogear-ups-103.md (hunk 2 above)
Comment (matzew) [inline _posts/2015-03-23-aerogear-ups-103.md]: it is optional. if you want to upgrade, see link for details

On Monday, March 23, 2015, Lukáš Fryč <notifications@github.com> wrote:

> In _posts/2015-03-23-aerogear-ups-103.md
> <https://github.com/aerogear/aerogear.org/pull/566#discussion_r26924747>:
>
> > @@ -0,0 +1,25 @@
> > +---
> > +layout: news
> > +section: news
> > +title: AeroGear UnifiedPush Server 1.0.3 released
> > +author: matzew
> > +module: push
> > +releases: 1.0.3
> > +---
> > +
> > +The AeroGear team is happy to announce the version 1.0.3 of the UnifiedPush Server! The release focuses on stabilization and adds a few new features
> > +
> > +* Migration support for 1.0.x users as explained in our [user guide](https://aerogear.org/docs/unifiedpush/ups_userguide/index/#migration-guide)
>
> Is migration a feature?
>
> If yes, I would start with the features that caused a need for migration
> and then explained they unfortunately requires database migration.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear.org/pull/566/files#r26924747>.
>


-- 
Sent from Gmail Mobile

Comment (lfryc) [inline _posts/2015-03-23-aerogear-ups-103.md]: I will send a suggestion for change in a PR against this one.

Comment (matzew) [inline _posts/2015-03-23-aerogear-ups-103.md]: +1

On Tue, Mar 24, 2015 at 4:12 PM, Lukáš Fryč <notifications@github.com>
wrote:

> In _posts/2015-03-23-aerogear-ups-103.md
> <https://github.com/aerogear/aerogear.org/pull/566#discussion_r27036989>:
>
> > @@ -0,0 +1,25 @@
> > +---
> > +layout: news
> > +section: news
> > +title: AeroGear UnifiedPush Server 1.0.3 released
> > +author: matzew
> > +module: push
> > +releases: 1.0.3
> > +---
> > +
> > +The AeroGear team is happy to announce the version 1.0.3 of the UnifiedPush Server! The release focuses on stabilization and adds a few new features
> > +
> > +* Migration support for 1.0.x users as explained in our [user guide](https://aerogear.org/docs/unifiedpush/ups_userguide/index/#migration-guide)
>
> I will send a suggestion for change in a PR against this one.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear.org/pull/566/files#r27036989>.
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

Comment (lfryc) [inline _posts/2015-03-23-aerogear-ups-103.md]: Here it is: https://github.com/matzew/aerogear.org/pull/11

**Comment by lfryc** [comment]
Merged and :rocket:  to production.

**Pull request merged**